colors.go

```diff
@@ -82,11 +82,19 @@ func parseStyles() styleMap {
 }
 
 func parseEscapeSequence(s string) tcell.Style {
+	return parseApplyEscapeSequence(s, tcell.StyleDefault)
+}
+
+func parseApplyEscapeSequence(s string, st tcell.Style) tcell.Style {
```

Comment:
`applyParsedEscapeSequence` might be slightly better.

Reply:
Fixed in 22eda30